cpp/src/arrow/compute/exec/CMakeLists.txt

```diff
@@ -32,8 +32,8 @@ add_arrow_compute_test(util_test PREFIX "arrow-compute")
 
 add_arrow_benchmark(expression_benchmark PREFIX "arrow-compute")
 
-find_package(OpenMP)
-if(OpenMP_CXX_FOUND)
+if(ARROW_BUILD_OPENMP_BENCHMARKS)
```

Comment:
Can you add this too `CMakePresets.json` (base-benchmarks) and `DefineOptions.cmake`?

Reply:
Done! Please double check it though since I didn't really know what I was doing modifying that file.